docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| 🇳🇴 Norway               | [MET Norway](#met-norway)                          | ~10 days |
+| 🇵🇭 Philippines          | [PAGASA](#pagasa)                                  | 5 days   |
+| 🇵🇹 Portugal             | [IPMA](#instituto-português-do-mar-e-da-atmosfera) | 10 days  |
+| 🇵🇷 Puerto Rico          | [NWS](#national-weather-service)                   | 7 days   |
+| 🇸🇲 San Marino           | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+| 🇸🇯 Svalbard & Jan Mayen | [MET Norway](#met-norway)                          | ~10 days |
+| 🇸🇪 Sweden               | [SMHI](#smhi)                                      | 15 days  |
+| 🇹🇼 Taiwan               | [CWA](#central-weather-administration) 🔐          | 7 days   |
+| 🇹🇷 Türkiye              | [MGM](#meteoroloji-genel-müdürlüğü)                | 5 days   |
+| 🇬🇧 United Kingdom       | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇺🇸 United States        | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇮 U.S. Virgin Is.      | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇦 Vatican City         | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+
+## Worldwide sources
+
+### Open-Meteo
+**[Open-Meteo](https://open-meteo.com/)** is a weather data provider based in Bürglen, Switzerland. It is the only free and open source weather source on this list, and probably also the most privacy-friendly. When we don’t support your national weather source, we suggest using **Open-Meteo** as your primary weather source.
+
+| Feature                        | Detail                                                                           |
+|--------------------------------|----------------------------------------------------------------------------------|
+| 🔐 **API key**                 | Not required                                                                     |
+| 🗺️ **Coverage**               | 🌐 Worldwide (some features may not be available for all locations)              |
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station           |
+| 📆 **Daily forecast**          | Up to 15 days                                                                    |
+| ⏱️ **Hourly forecast**         | Up to 16 days                                                                    |
+| 😶‍🌫️ **Air quality**         | Available                                                                        |
+| 🤧 **Pollen**                  | Available in Europe for: Alder, Birch, Grass, Mugwort, Olive, and Ragweed        |
+| ☔ **Precipitation nowcasting** | Available (works best in Europe at the moment)                                   |
+| ⚠️ **Alerts**                  | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
+| 📊 **Normals**                 | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
```

Comment:
I think we should remove suggestions to use other sources in this area, as:
- it is already recommended at the beginning of the doc
- some users may not have access to AccuWeather in the free-network-only version
- we may change our recommendation any time, leading to more maintenance of the cost if the information is repeated
- the actual recommendation is to use your national source if available

Reply:
There are still many sources mentioning other recommended sources

Reply:
Ah! I read it as though you had just wanted to remove them from Open-Meteo. I can remove all secondary source recommendations from all main sources.